bson/src/main/org/bson/codecs/pojo/CreatorExecutable.java

```diff
@@ -137,7 +137,9 @@ CodecConfigurationException getError(final Class<?> clazz, final String msg) {
 
     private void checkHasAnExecutable() {
         if (constructor == null && method == null) {
-            throw new CodecConfigurationException(format("Cannot find a public constructor for '%s'.", clazz.getSimpleName()));
+            throw new CodecConfigurationException(format("Cannot find a public constructor for '%s'.%n"
```

Comment:
This is causing `org.bson.codecs.pojo.PojoCustomTest.testConstructorNotPublicModel` to fail with the exception `java.util.MissingFormatArgumentException: Format specifier '%s'`, because there are not enough arguments.  I don't see a need to repeat the class name (or a newline for that matter), so I think it would be fine to just have:

     format("Cannot find a public constructor for '%s'.  "Please ensure the class has a public, empty constructor with no arguments, clazz.getSimpleName())